scapy/layers/l2.py

```diff
@@ -549,6 +549,366 @@ class EAP_FAST(Packet):
     ]
 
 
+#############################################################################
+##### IEEE 802.1X-2010 - MACsec Key Agreement (MKA) protocol
+#############################################################################
+
+#________________________________________________________________________
+#
+# IEEE 802.1X-2010 standard
+# Section 11.11.1
+#________________________________________________________________________
+#
+
+_parameter_set_types = {
+    1:   "Live Peer List",
+    2:   "Potential Peer List",
+    3:   "MACsec SAK Use",
+    4:   "Distributed SAK",
+    5:   "Distributed CAK",
+    6:   "KMD",
+    7:   "Announcement",
+    255: "ICV Indicator"
+}
+
+
+# Used by MKAParamSet::dispatch_hook() to instantiate the appropriate class
+_param_set_cls = {
+    1:   "MKALivePeerListParamSet",
+    2:   "MKAPotentialPeerListParamSet",
+    3:   "MKASAKUseParamSet",
+    4:   "MKADistributedSAKParamSet",
+    255: "MKAICVSet",
+}
+
+
+class MACsecSCI(Packet):
+    """
+    Secure Channel Identifier.
+    """
+
+    #________________________________________________________________________
+    #
+    # IEEE 802.1AE-2006 standard
+    # Section 9.9
+    #________________________________________________________________________
+    #
+
+    name = "SCI"
+    fields_desc = [
+        MACField("system_identifier", "00:00:00:00:00:00"),
```

Comment:
Maybe you could use `SourceMACField("system_identifier", None)` to get a "default value that works"?